automates/model_assembly/networks.py

```diff
@@ -1079,9 +1135,10 @@ def _(
             live_variables: Dict[VariableIdentifier, VariableNode],
             subgraph: GrFNSubgraph,
         ) -> None:
-            inputs = [live_variables[id] for id in stmt.inputs]
-            out_nodes = [add_variable_node(var) for var in stmt.outputs]
-            func = add_lambda_node(stmt.type, stmt.func_str)
+            inputs = [variable_nodes[v_id] for v_id in stmt.inputs]
+            out_nodes = [variable_nodes[v_id] for v_id in stmt.outputs]
+            # out_nodes = [add_variable_node(var) for var in stmt.outputs]
```

Comment:
can probably delete this

Reply:
I can't find this one so I guess someone else deleted it in some other branch that was then merged into this branch via master.